src/zjs_common.json

```diff
@@ -2,10 +2,12 @@
     "module": "common",
     "zephyr_conf": {
         "all": [
+            "CONFIG_CPLUSPLUS=y",
             "CONFIG_STDOUT_CONSOLE=y",
             "CONFIG_NEWLIB_LIBC=y",
             "CONFIG_FLOAT=y",
             "CONFIG_RING_BUFFER=y"
         ]
-    }
+    },
+    "zjs_config": ["-DENABLE_INIT_FINI"]
```

Comment:
isn't this a linux only config?

Reply:
No its needed in every version in order to enable JerryScript modules.